rpg-custom-rules/README.md

```diff
@@ -2,8 +2,8 @@
 
 This example demonstrates how to write **Custom Rules** for SonarRPG.
 
-See also the [documentation on how to write custom rules for RPG](http://docs.sonarqube.org/display/PLUG/Custom+Rules+for+RPG).
+See also the [documentation on how to write custom rules for RPG](https://docs.sonarsource.com/sonarqube/latest/analyzing-source-code/languages/rpg/).
 
 This example assumes that the jar file of the RPG plugin is available in the `lib` directory. 
-You can download sonar-rpg-plugin-2.2.0.1005.jar 
+You can download sonar-rpg-plugin-3.9.0.5001.jar
```

Comment:
I don't think we can download the JAR from the SonarSource website anymore (I can't find it at least :smile:). Maybe we can use the link that is in the `build.sh` file (https://binaries.sonarsource.com/CommercialDistribution/sonar-rpg-plugin/sonar-rpg-plugin-3.9.0.5001.jar) here.